app/main.py

```diff
@@ -1,2 +1,53 @@
+from typing import Hashable
+
+
 class Dictionary:
-    pass
+    DEFAULT_CAPACITY = 8
+    LOAD_FACTOR = 2 / 3.0
```

Comment:
The `LOAD_FACTOR` is incorrectly calculated as `2 / 3.0`, which results in a float division. It should be `2 / 3` or `2.0 / 3.0` to ensure the result is a float.